requirements.txt

```diff
@@ -1,4 +1,4 @@
-black==20.8b1
+black==21.12b0
```

Comment:
I had to bump `black`'s version, or else the `black` check wouldn't pass for Python > 3.9.8, cf. https://stackoverflow.com/q/69912264

Reply:
+1